services/src/main/java/org/keycloak/truststore/TruststoreBuilder.java

```diff
@@ -48,9 +49,17 @@ public class TruststoreBuilder {
     public static final String DUMMY_PASSWORD = "keycloakchangeit"; // fips length compliant dummy password
     static final String PKCS12 = "PKCS12";
 
+    private static final String DEFAULT_KUBERNETES_CA_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt";
+    private static final String DEFAULT_SERVICE_CA_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt";
+    // Testability and flexibility: allow overriding CA paths via system properties
```

Comment:
I don't see this being used by tests. As for flexibility, wouldn't users just add a truststore path, rather than set a system property to change the default kubernetes locations?

Reply:
It's a leftover. Removing.